builder/builder.pb.go

```diff
@@ -0,0 +1,611 @@
+// Copyright 2018 Google LLC
```

Comment:
Is checking in the generated code actually the recommended Thing To Do?

Reply:
Yes, other packages do the same.